corehq/apps/accounting/models.py

```diff
@@ -371,6 +372,12 @@ class BillingAccount(ValidateModelMixin, models.Model):
         choices=PreOrPostPay.CHOICES,
     )
 
+    # Settings visible to external users
+    restrict_domain_creation = models.BooleanField(default=False)
```

Comment:
The background on this feature is that it's for clients who have many domains and need tools to manage them in bulk. So it's set at the account level since it isn't feasible to set it on each individual domain page (and also on new domains).